templates/components/common/navigation-menu.html

```diff
@@ -2,231 +2,96 @@
+                    {{#if ../theme_settings.navigation_design '===' 'simple'}}
+                        {{> components/common/navigation-list}}
+                    {{/if}}
+                    {{#if ../theme_settings.navigation_design '===' 'alternate'}}
+                        {{> components/common/navigation-list-custom categories=../categories social_media=../social_media }}
+                    {{/if}}
+                </li>
+                <svg class="c-navigation__separator" xmlns="http://www.w3.org/2000/svg" width="25" height="24" viewBox="0 0 25 24" fill="none">
+                    <path d="M13.5 12C13.5 12.5523 13.0523 13 12.5 13C11.9477 13 11.5 12.5523 11.5 12C11.5 11.4477 11.9477 11 12.5 11C13.0523 11 13.5 11.4477 13.5 12Z" stroke="#E6DABB" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+                </svg>
+            {{/unless}}
         {{/each}}
-        {{#unless theme_settings.hide_content_navigation}}
-            {{#each pages}}
+    </ul>
+    <div class="c-navigation__megamenu is-mobile">
+        <div class="js-quick-search">
+            {{> components/common/quick-search name='nav-menu-quick-search'}}
+        </div>
+        <ul class="c-navigation-list__submenu">
+            {{#each categories}}
+                {{#each children}}
+                    <li>
+                        {{#if children}}
+                            <a 
+                                class="js-nav-pages-action has-sub-menu{{#if is_active}} is-active{{/if}} c-navigation__link c-navigation-list__submenu-header"
+                                href="{{url}}"
+                                aria-label="{{name}}"
+                            >
+                                {{name}}
+                            </a>
+                            <ul id="nav-pages-{{id}}" class="is-open c-navigation-list__submenu-list">
+                                {{#each children}}
+                                    <li>
+                                        <a 
+                                            class="js-nav-pages-action{{#if is_active}} is-active{{/if}} c-navigation__link c-navigation-list__submenu-item"
+                                            href="{{url}}"
+                                            aria-label="{{name}}"
+                                        >
+                                            {{name}}
+                                        </a>
+                                    </li>
+                                {{/each}}
+                            </ul>
+                        {{/if}}
+                    </li>
+                {{/each}}
+            {{/each}}
+            <ul class="c-navigation-list__submenu-list">
                 <li>
                     <a 
-                        class="js-nav-pages-action{{#if name '==' ../page.title}} active-page{{/if}}"
+                        class="js-nav-pages-action c-navigation-list__submenu-header"
```

Comment:
Maybe update name to the component's name e.g. `c-navigation-menu__submenu-header`?

Reply:
It's used in different components